demos/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ include_directories(
 
 set(DEMOS_SCRIPTS
     task_space_control_loop
+    kinematics_control_loop
```

Comment:
Inconsistency between CMakeLists and actual filename. Does not build.

Reply:
Indeed I renamed it just in the last push and because it is not added normally clion did not refactor. Good catch